src/VisualStudio/Core/Def/ValueTracking/ValueTrackingCommandHandler.cs

```diff
@@ -144,6 +143,12 @@ private async Task ShowToolWindowAsync(ITextView textView, ISymbol selectedSymbo
                 return;
             }
 
+            var toolWindow = await GetOrCreateToolWindowAsync(textView, cancellationToken).ConfigureAwait(false);
+            if (toolWindow is null || toolWindow.ViewModel is null)
```

Comment:
can this happen?

Reply:
It shouldn't, but at most we should log a problem and not crash here imo

Reply:
my request is that we either log, or doc this code.  right now it looks like we're just silent.  which makes me feel like a customer will try to use this feature, see nothing appear and have no idea what's going on :)